BUILD.bazel

```diff
@@ -501,7 +506,7 @@ cc_library(
     deps = [
         ":IlmThread",
         ":OpenEXRCore",
-        "@imath//:Imath",
```

Comment:
Module names are typically written in lowercase. See for example [here](https://github.com/bazelbuild/bazel-central-registry/tree/main/modules) - no module is written in uppercase. I think this is done to prevent OS problems - for instance Windows vs. Linux - Windows would maybe treat imath, Imath, IMATH the same way. Change it back to` "@imath//:Imath",`

Reply:
I agree, except that openexr uses a reference to the imath bazel module dep that capitalizes it.

Reply:
In the bazel central registry that is.

Reply:
eg: https://github.com/bazelbuild/bazel-central-registry/blob/main/modules/openexr/3.2.4/MODULE.bazel#L11